skins/oasis/css/modules/ContentReview.scss

```diff
@@ -48,8 +48,28 @@ $content-review-color-unsubmitted: #ffad00;
 
 .content-review-status-unsubmitted {
 	border-color: $content-review-color-unsubmitted;
+	animation: submitReminder 3s infinite;
 }
 
 .content-review-status-awaiting {
 	border-color: $content-review-color-awaiting;
 }
+
+/* Animation */
+@keyframes submitReminder {
```

Comment:
@adamkarminski @hakubo per js fiddle visualization, I think it could sound well if you start a band together ;) very rhythmic visualization :)